include/SDL3/SDL_hints.h

```diff
@@ -2499,6 +2499,16 @@ extern "C" {
  */
 #define SDL_HINT_GDK_TEXTINPUT_MAX_LENGTH "SDL_GDK_TEXTINPUT_MAX_LENGTH"
 
+/**
+ *  \brief  A variable controlling whether the Metal render driver select low power device over default one
+ *
+ *  This variable can be set to the following values:
+ *    "0"       - Use the prefered OS device
+ *    "1"       - Select a low power one
+ *
+ *  By default the prefered OS device is used.
+ */
+#define SDL_HINT_METAL_PREFER_LOW_POWER_DEVICE "SDL_METAL_PREFER_LOW_POWER_DEVICE"
```

Comment:
Can you make this SDL_HINT_METAL_... and group it with the other renderer hints?

Thanks!